k8s/conf/vpn-gateway-nse.yaml

```diff
@@ -36,7 +36,7 @@ spec:
             - name: TRACER_ENABLED
               value: "true"
             - name: IP_ADDRESS
-              value: "10.60.1.0/24"
+              value: "172.16.1.0/24"
```

Comment:
Why this change?

Reply:
To exclude case with IP intersections on clouds. Because by logs which I saw looks like it is a root cause. Also, I've expanded logs for bash tests to get more information about the problem if it is still reproducible.

Reply:
Now bash tests logs save on CI in artifacts directory.